ingestion/tests/integration/cockroach/conftest.py

```diff
@@ -0,0 +1,66 @@
+import os
+import textwrap
+
+import pytest
+from sqlalchemy import create_engine
+from testcontainers.cockroachdb import CockroachDBContainer
+
+from _openmetadata_testutils.helpers.docker import try_bind
+from metadata.generated.schema.api.services.createDatabaseService import (
+    CreateDatabaseServiceRequest,
+)
+from metadata.generated.schema.entity.services.connections.database.cockroachConnection import (
+    CockroachConnection,
+)
+from metadata.generated.schema.entity.services.databaseService import (
+    DatabaseConnection,
+    DatabaseServiceType,
+)
+
+
+@pytest.fixture(scope="module")
+def cockroach_container(tmp_path_factory):
+    """
+    Start a Cockroach container with the dvdrental database.
+    """
+
+    container = CockroachDBContainer(
+        image="cockroachdb/cockroach:v23.1.0", dbname="dvdrental"
```

Comment:
curiosity: why `dvdrental`? Does the image come with preloaded data?

Reply:
That was a mistake from my side, resolved it